package.json

```diff
@@ -70,7 +70,7 @@
     "transaction"
   ],
   "peerDependencies": {
-    "react": "~17.0"
+    "react": "17.0.2"
```

Comment:
let's make it ~, so we don't depend on an absolute version

Reply:
Done